pom.xml

```diff
@@ -8,7 +8,7 @@
         <version>22</version>
     </parent>
 
-    <groupId>io.takari.maven.plugins</groupId>
+    <groupId>com.facebook.presto</groupId>
```

Comment:
It's not related, I just asked him to throw it in here as a separate commit to make it easier. `takari-maven-plugin` comes from the Takari lifecycle defined in the base POM. It's fairly well documented now in this free book: http://takari.io/book/